src/components/MapView/DateSelector/index.tsx

```diff
@@ -124,11 +120,12 @@ function DateSelector({
       x: dateIndex * TIMELINE_ITEM_WIDTH,
       y: 0,
     });
-    setSelectedDate(moment(stateStartDate));
```

Comment:
I was really confused why we are managing two variables both tracking the same thing (stateSelectedDate). I removed the state in favor of redux to reduce coupling and make things easier to read/track. Let me know if this was done purposefully.

Reply:
(should test thoroughly to make sure nothing is broken with this refactoring)